Corona-Warn-App/src/main/java/de/rki/coronawarnapp/environment/BuildConfigWrap.kt

```diff
@@ -0,0 +1,12 @@
+package de.rki.coronawarnapp.environment
+
+import de.rki.coronawarnapp.BuildConfig
+
+// Can't be const because that prevents them from being mocked in tests
+@Suppress("MayBeConstant")
+object BuildConfigWrap {
```

Comment:
Just for clarification: You introduce a Wrapper so the config can be injected and mocked ?

Reply:
Correct, for mocking. Tooling for mocking static java stuff is really cumbersome (`BuildConfig.java`).